client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -86,6 +113,18 @@ class StatsPostDetail extends Component {
 				/>
 
 				<PostWeeks siteId={ this.props.siteId } postId={ this.props.postId } />
+				<WebPreview
```

Comment:
Indeed, I should have disclosed that in the PR summary.  I had chatted with @supernovia about that when we discussed this feature in slack.  The exposure here is a bit _less_ than in the editor, as posts in stats have been published.

Granted there could still be issues using the web preview with private posts on JP sites, and the user not being logged in there... but it won't always fail like viewing a draft on a JP site does.